### PR TITLE
chore: run gi tests in the CI

### PR DESCRIPTION
**Pull request by bdraco**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -46,14 +46,19 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Install libs
-        run: sudo apt-get install -y dbus dbus-x11 python-gi
```
Comment (dlech) [inline .github/workflows/ci.yml:49]: This should probably just be `python3-gi`. `python-gi` would install the Python 2 version.

Comment (bdraco) [reply .github/workflows/ci.yml:49]: Let's give that a shot 👍

Comment (bdraco) [reply .github/workflows/ci.yml:49]: Looks like they still get skipped since its not for the right python version

**Comment by bdraco** [comment]
Still looks good this morning 😄

**Pull request merged**